lms/djangoapps/verify_student/models.py

```diff
@@ -93,6 +93,10 @@ class IDVerificationAttempt(StatusModel):
     Each IDVerificationAttempt represents a Student's attempt to establish
     their identity through one of several methods that inherit from this Model,
     including PhotoVerification and SSOVerification.
+
+    .. pii: The User's name is stored in this and sub-models
+    .. pii_types: name
+    .. pii_retirement: retained
```

Comment:
This abstract model technically doesn't need an annotation.  I'm fine with annotating it, but keep it mind it may or may not end up in reports depending on whether the reporting script looks at all models, or just models requiring annotations.

Reply:
It should show up in anything that inherits from it as well, so that way if people write models that inherit but forget to annotate for the parent model it will still be in the report.